index.html

```diff
@@ -61,7 +61,7 @@ <h3 id="pure-wheel">Pure Python</h3>
 </pre>
                     <h3 id="dirty-wheel">C extensions</h3>
                     <p>PyPI currently allows uploading platform-specific wheels for Windows, macOS and Linux. It is useful to create wheels for these platforms, as it avoids the need for your users to compile the package when installing.</p>
-                    <p>You will need to have access to the platform you are building for.</p>
+                    <p>You will need to have access to the platform you are building for. Projects like <a href="https://github.com/matthew-brett/multibuild">multibuild</a> or <a href="https://github.com/joerick/cibuildwheel">cibuildwheel</a> provide tooling and guidance to build binary wheels for different platforms using various continuous integration (CI) services.</p>
```

Comment:
Thank you for opening this PR. Apologies that it's taken so long for me to get to this.

I notice that [`matthew-brett/multibuild`](https://github.com/matthew-brett/multibuild) seems to be a less active fork of [`multibuild/multibuild`](https://github.com/multibuild/multibuild). I wonder if we should now reference `multibuild/multibuild` instead?

Given the time that's elapsed since this PR was opened, are both of these projects still the best two to link to?

Reply:
Yes, https://github.com/matthew-brett/multibuild was the original home, it since moved to https://github.com/multibuild/multibuild (and the original author now has his personal fork).

And yes, I think they are still the best to link to, possibly with cibuldwheel first?

Reply:
Thanks! I'll apply these changes then, and merge:

```suggestion
                    <p>You will need to have access to the platform you are building for. Projects like <a href="https://github.com/pypa/cibuildwheel">cibuildwheel</a> or <a href="https://github.com/multibuild/multibuild">multibuild</a> provide tooling and guidance to build binary wheels for different platforms using various continuous integration (CI) services.</p>
```